style.css

```diff
@@ -40,6 +40,41 @@ a:active {
   margin: 0 1.5rem;
 }
 
+.card h1 {
+  /* width: auto;
+  height: 57px; */
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
```

Comment:
Please kindly remove the comment if not in use, it does not make your code look neat